src/lib/signature_lib/test/signature_lib_tests.ml

```diff
@@ -0,0 +1,39 @@
+open Signature_lib
```

Comment:
my understanding is that there is no global `dune runtest` running in CI, am I mistaken?

Reply:
`dune runtest src/lib/` is run, among others.

Reply:
ah I was wrong then